networkx/algorithms/distance_measures.py

```diff
@@ -452,7 +459,6 @@ def harmonic_diameter(G, sp=None):
         return math.inf
     return math.nan
 
-
```

Comment:
This has broken the formatter - see the [contributing guide](https://networkx.org/documentation/latest/developer/contribute.html) for info on how to automatically conform to the project's formatting rules.